recipes/BEHST/meta.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: behst
+  version: 29efb4c3a8e333a2824430ef32d8bf01
+
+source:
+  fn: 326f13179c55.zip
+  url: https://bitbucket.org/hoffmanlab/behst/get/326f13179c55.zip
+  md5: 59b4e4ede8d1362ccc2de312d8f39421
+
+build:
+  number: 1
+  skip: True # [not py27]
```

Comment:
This does probably not work, you only can use one skip: True. But you can write something like [not py27 or osx]